src/analysis/static/TargetPool.ts

```diff
@@ -0,0 +1,37 @@
+import { TargetFile } from "./TargetFile";
+import { CFG } from "./graph/CFG";
+
+export abstract class TargetPool {
+  private _included: TargetFile[];
```

Comment:
I think it would be clearer if we only store the targets not which are included and excluded. How the input from the CLI is translated to the actual targets can be extracted into a method or strategy pattern

Given, we are storing information on the actual target classes now, it might be better to just store the target classes themselves. We don't need to lazy load them since all of them will be tested anyway.

Reply:
Done